examples/tensorflow_simple.py

```diff
@@ -0,0 +1,148 @@
+"""
+Optuna example that optimizes multi-layer perceptrons using Tensorflow.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+Tensorflow and MNIST. We optimize the neural network architecture as well as the optimizer
+configuration. As it is too time consuming to use the whole MNIST dataset, we here use a small
+subset of it.
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python tensorflow_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --storage sqlite:///example.db`
+    $ optuna study optimize tensorflow_simple.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import shutil
+import tensorflow as tf
+
+import optuna
+
+MODEL_DIR = "/tmp/mnist_convnet_model"
+BATCH_SIZE = 128
+TRAIN_STEPS = 1000
+
+
+def create_network(trial, features):
+    # We optimize the numbers of layers and their units.
+    input_layer = tf.reshape(features['x'], [-1, 784])
+    prev_layer = input_layer
+
+    n_layers = trial.suggest_int('n_layers', 1, 3)
+    for i in range(n_layers):
+        n_units = int(trial.suggest_int('n_units_l{}'.format(i), 1, 128))
```

Comment:
This type conversion is redundant because `trial.suggest_int` returns integer values.